doc/translation_strings_policy.md

```diff
@@ -33,25 +33,25 @@ General recommendations
 
 Try not to burden translators with translating messages that are e.g. slight variations of other messages.
 In the GUI, avoid the use of text where an icon or symbol will do.
-Make sure that placeholder texts in forms don't end up in the list of strings to be translated (use `<string notr="true">`).
+Make sure that placeholder texts in forms do not end up in the list of strings to be translated (use `<string notr="true">`).
 
 ### Make translated strings understandable
 
-Try to write translation strings in an understandable way, for both the user and the translator. Avoid overly technical or detailed messages
+Try to write translation strings in an understandable way, for both the user and the translator. Avoid overly technical or detailed messages.
 
 ### Do not translate internal errors
 
-Do not translate internal errors, or log messages, or messages that appear on the RPC interface. If an error is to be shown to the user,
-use a translatable generic message, then log the detailed message to the log. E.g. "A fatal internal error occurred, see debug.log for details".
+Do not translate internal errors, log messages, or messages that appear on the RPC interface. If an error is to be shown to the user,
+use a translatable generic message, then log the detailed message to the log. E.g "A fatal internal error occurred, see debug.log for details".
```

Comment:
"E.g." should be followed by a comma (as we use American English for docs).
Ref: https://en.wikipedia.org/wiki/List_of_Latin_phrases_(E)#e.g.

Reply:
good catch, will update!